webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -28,10 +28,9 @@ const Breadcrumb = ({
     <PfBreadcrumb {...props}>
       {items.map((item, index) => {
         const active = index === items.length - 1;
-        const {
-          caption,
-          caption: { icon, text },
-        } = item;
+        const { caption, url, onClick } = item;
+        const { icon, text } = caption || {};
+
         const overrideTitle = active && titleReplacement;
         const itemTitle = overrideTitle || text || caption;
```

Comment:
Can you add here `|| ''`?
`EllipsisWithTooltip` shows an error when it's null

Reply:
Thanks, updated. @amirfefer - would be good to update `EllipsisWithTooltip` to properly handle that case as well.